airbyte-workers/src/main/java/io/airbyte/workers/WorkerUtils.java

```diff
@@ -100,18 +100,19 @@ public static WorkerDestinationConfig syncToWorkerDestinationConfig(final Standa
   }
 
   public static Map<String, JsonNode> mapStreamNamesToSchemas(final StandardSyncInput syncInput) {
-    final String streamPrefix = syncInput.getPrefix();
     return syncInput.getCatalog().getStreams().stream().collect(
         Collectors.toMap(
             k -> {
-              final String namespace = Objects.toString(k.getStream().getNamespace(), "").trim();
-              final String name = k.getStream().getName().trim();
-              return namespace + name;
+              return streamNameWithNamespace(k.getStream().getNamespace(), k.getStream().getName());
             },
             v -> v.getStream().getJsonSchema()));
 
   }
 
+  public static String streamNameWithNamespace(final String namespace, final String streamName) {
```

Comment:
nit: add a `@Nullable` annotation in front of `String namespace`, since we expect that to be nullable in some cases